### DISCUSSION TITLE
Ex2 Part 1 - Different ids for same text in query/doc

### DISCUSSION THREAD

**Discussion by CorpaciLC** [sebastian-hofstaetter/teaching#90] [Advanced IR: Exercise Q&A] [answered]
Hi,

We saw that there are different ids for the same query texts in some cases (e.g. "financial times" ). Same happens for some pairs of doc ids and doc texts.

![image](https://user-images.githubusercontent.com/16256637/170883549-19521d90-2695-436b-be37-f2d929dcaf7a.png)

Could we aggregate all the ids for one such text sample? This would increase the number of judgements we have per pair query_id <-> doc_id.

Best,
Group 11

**Comment by sophiaalthammer** [comment, marked as answer]
Hi,

thats a good and interesting finding! But I would not recommend aggregating the query ids, so that the test collections to the other teams remain comparable!

Best,
Sophia

**Reply by LaurenzV** [reply]
Our team also aggregated the same queries/documents, but we've already done the evaluation of the three models... Is this a big issue and will it lead to point deductions if we leave it like that?
